index.html

```diff
@@ -348,12 +348,12 @@
                         <thead><tr><th>Device Button Changes</th><th><code>button</code></th></tr></thead>
                         <tbody>
                             <tr><td>Neither buttons nor touch/pen contact changed since last event</td><td>-1</td></tr>
-                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact (with no modifier buttons pressed)</td><td>0</td></tr>
+                            <tr><td>Left Mouse,<br>Touch Contact,<br>Pen contact</td><td>0</td></tr>
```

Comment:
Random question: are there pens that have a (redundant/additional) "left click" button which would allow a hovering click? If so, we'd need to probably add that here too, unless that makes it too confusing?